src/UI/UIView.cpp

```diff
@@ -452,16 +452,16 @@ void UIView::draw_ui_callback()
           m_stopped = !m_stopped;
         }
         ImGui::SameLine();
-        int value = int(m_animation_speed * 100.0 + .5);
+        int value = int(m_animation_step * 100.0 + .5);
         std::string speed_text = ">>X " + std::to_string(value / 100);
         if (ImGui::Button(speed_text.c_str(), ImVec2(80, 0)))
         {
-          m_animation_speed += 0.5;
-          if (m_animation_speed > 4.0f)
+          m_animation_step += 0.5;
```

Comment:
why a half step?

Reply:
just number I came up with. This is related to the volume animation, not camera animation.